tests/posix/net/testcase.yaml

```diff
@@ -0,0 +1,10 @@
+common:
+  tags:
+    - iface
+    - net
+    - posix
+  depends_on: netif
+  arch_exclude:
```

Comment:
I found this was necessary because `native_sim` requires super user privileges (at least on my desk).

Can be fixed later, most likely